### PR TITLE
mention impact of VO on job priority

### PR DESCRIPTION
**Pull request by boegel**
Note: this whole section is under `\ifgent` already.

### CODE REVIEW

File: intro-HPC/ch_running_batch_jobs.tex [hunk 1]
```diff
@@ -773,19 +773,37 @@ \subsection{When will my job start?}
...
+    \item Historical use: the aim is to balance usage over users, so
         infrequent (in terms of total compute time used) users get a higher priority
 
-    \item requested resources (amount of cores, walltime, memory, \ldots)
+    \item Requested resources (amount of cores, walltime, memory, \ldots).
+        The more resources you request, the more likely it is the job(s) will
+        have to wait for a while until those resources become available.
 
-    \item time waiting in queue: queued jobs get a higher priority over time
+    \item Time waiting in queue: queued jobs get a higher priority over time.
```
Comment (itkovian) [inline intro-HPC/ch_running_batch_jobs.tex:785]: Mention there is an upper limit to the contribution.

File: intro-HPC/ch_running_batch_jobs.tex [hunk 2]
```diff
@@ -773,19 +773,37 @@ \subsection{When will my job start?}
...
 
-    \item user limits: this avoids having a single user use the entire cluster.
+    \item User limits: this avoids having a single user use the entire cluster.
```
Comment (itkovian) [inline intro-HPC/ch_running_batch_jobs.tex:787]: Technically, this is no longer correct, and if we want it back, we can, but we should decide on it then :)

File: intro-HPC/ch_running_batch_jobs.tex [hunk 3]
```diff
@@ -773,19 +773,37 @@ \subsection{When will my job start?}
...
         This means that each user can only use a part of the cluster.
 
+    \item Whether or not you are a member of a \emph{Virtual Organisation} (VO).
+
+        Each VO gets assigned a \emph{fair share target}, which has a big impact
+        on the job priority. This is done to let the job scheduler balance usage
+        across different research groups.
+
+        If you are not a member of a specific VO, you are sharing a fair share target
```
Comment (itkovian) [inline intro-HPC/ch_running_batch_jobs.tex:796]: Technically, the default VO is also specific :p

File: intro-HPC/ch_running_batch_jobs.tex [hunk 4]
```diff
@@ -773,19 +773,37 @@ \subsection{When will my job start?}
...
+the priority of the job, which is determined by:
 \begin{itemize}
-    \item historical use: the aim is to balance usage over users, so
+    \item Historical use: the aim is to balance usage over users, so
         infrequent (in terms of total compute time used) users get a higher priority
 
-    \item requested resources (amount of cores, walltime, memory, \ldots)
+    \item Requested resources (amount of cores, walltime, memory, \ldots).
+        The more resources you request, the more likely it is the job(s) will
+        have to wait for a while until those resources become available.
```
Comment (itkovian) [inline intro-HPC/ch_running_batch_jobs.tex:783]: Maybe also mention the need to pack the jobs as much as possible?

**Comment by alvarosimon** [comment]
fixed by #508

**Pull request closed without merging**